starter-code/app.js

```diff
@@ -11,7 +11,7 @@ const path         = require('path');
 
 
 mongoose
-  .connect('mongodb://localhost/starter-code', {useNewUrlParser: true})
+  .connect('mongodb://localhost/Express-movies', {useNewUrlParser: true})
```

Comment:
Remember to always add using unifiedTopology for mongoose connection. 
```suggestion
  .connect('mongodb://localhost/Express-movies', {useNewUrlParser: true, useUnifiedTopology: true})
```